mgmt/rest/task.go

```diff
@@ -66,6 +66,7 @@ type task struct {
 	FailedCount        uint                    `json:"failed_count,omitempty"`
 	LastFailureMessage string                  `json:"last_failure_message,omitempty"`
 	State              string                  `json:"task_state"`
+	MaxFailure         uint                    `json:"max_failure"`
```

Comment:
In the 2nd pull request will change that to uint.  Just try to make the minimum change here.